chain/chain/src/chain.rs

```diff
@@ -551,7 +551,13 @@ impl Chain {
     ) -> Result<(), Error> {
         let head = self.store.head()?;
         let tail = self.store.tail()?;
-        let gc_stop_height = self.runtime_adapter.get_gc_stop_height(&head.last_block_hash)?;
+        let gc_stop_height = match self.runtime_adapter.get_gc_stop_height(&head.last_block_hash) {
+            Ok(height) => height,
+            Err(e) => match e.kind() {
+                ErrorKind::DBNotFoundErr(_) => self.genesis.height(),
```

Comment:
`return Ok(())` because we can't go forward. Please add a comment describing what's happening here.